install.sh

```diff
@@ -135,6 +155,22 @@ if [[ $BUILD_ICU == yes ]]; then
         echo "Warning: --disable-shared recommended when building ICU."
     fi
 fi
+if [[ $BUILD_QRENCODE == yes ]]; then
+    if [[ !($PREFIX)]]; then    
+        echo "Warning: --prefix recommended when building QRENCODE."
+    fi
+    if [[ !($DISABLE_SHARED) ]]; then
+        echo "Warning: --disable-shared recommended when building QRENCODE."
+    fi
+fi
+if [[ $BUILD_LIBPNG == yes ]]; then
+    if [[ !($PREFIX)]]; then    
+        echo "Warning: --prefix recommended when building LIBPNG."
+    fi
+    if [[ !($DISABLE_SHARED) ]]; then
+        echo "Warning: --disable-shared recommended when building LIBPNG."
+    fi
+fi
```

Comment:
This is probably a good idea, although I'm not certain the same issues would require this precaution.